picard/tagger.py

```diff
@@ -605,12 +617,28 @@ def handle_command_save_modified(self, argstring):
     def handle_command_scan(self, argstring):
         self.analyze(self.unclustered_files.files)
 
+    def handle_command_set_saved_logs_verbosity(self, argstring):
+        if argstring.upper() in {"DEBUG", "D"}:
+            self.saved_logs_verbosity = logging.DEBUG
+        elif argstring.upper() in {"INFO", "I"}:
+            self.saved_logs_verbosity = logging.INFO
+        elif argstring.upper() in {"WARNING", "W"}:
+            self.saved_logs_verbosity = logging.WARNING
+        elif argstring.upper() in {"ERROR", "E"}:
+            self.saved_logs_verbosity = logging.ERROR
```

Comment:
I think we should left this verbosity thing out for now, and just dump the whole log.

Reply:
Disagreed, the command does not have a complex logic, yet it helps a lot with writing a clean log files.

Reply:
The thing is that what is logged depends on actual log level, so if Picard is set to INFO level, changing this to DEBUG will not change anything.
I don't say this isn't useful, but we need to think about how to implement it a bit more. What I suggest is to let this out this PR, and do that in an other.